### ISSUE TITLE
Write out Percentiles at end of test

### ISSUE THREAD

**Issue by maddisondavid** [pravega/pravega-benchmark#97]
Currently the test displays final statistics along with latency percentiles at the end of the test, however these are only written as long entries.  Some performance teams are using this information and currently using pattern matching to read and record these numbers.  This is time consuming and also extremely fragile and must be taken into consideration whenever the tools is modified.

Instead the final summary statistics should be written to a machine readable file for analysis.

**Comment by claudiofahey** [comment]
Consider using a format that is similar to the JSON file written by OpenMessaging Benchmark - https://github.com/pravega/openmessaging-benchmark/blob/dev/benchmark-framework/src/main/java/io/openmessaging/benchmark/TestResult.java.

**Comment by RaulGracia** [comment]
@maddisondavid I agree that an output file for results is a good idea. Moreover, I also noted that latency percentiles do not have decimal precision. In our case, it means that a value is either 2ms or 3ms, but it cannot be 2.5ms. For instance:
```
2020-01-22 16:48:48:540 +0000 [ForkJoinPool-1-worker-1] INFO io.pravega.perf.PerfStats - 5938 records Writing, 100.012 records/sec, 1000 bytes record size, 0.10 MiB/sec, 4.9 ms avg latency, 90.0 ms max latency, 5 ms 50th, 5 ms 75th, 6 ms 95th, 11 ms 99th, 15 ms 99.9th, 15 ms 99.99th.
```
As you can realize, in our scale this may be too coarse grained. Could we have decimal precision in percentiles too?

**Comment by claudiofahey** [comment]
Publish and end-to-end latency is currently measured at millisecond precision using System.currentTimeMillis(). Unlike the mean, percentiles can't be any more precise than the data.
